test/Microsoft.Health.Fhir.Ingest.UnitTests/Telemetry/ExceptionTelemetryProcessorTests.cs

```diff
@@ -20,6 +21,7 @@ public class ExceptionTelemetryProcessorTests
         [InlineData(typeof(NotSupportedException))]
         [InlineData(typeof(FhirResourceNotFoundException))]
         [InlineData(typeof(ResourceIdentityNotDefinedException))]
+        [InlineData(typeof(TemplateNotFoundException))]

```

Comment:
Should we change this to MappingFileNotFound?

Reply:
We have a lot of refactoring to do on that front :)